app/grants/templates/grants/card/clr_estimate.html

```diff
@@ -5,7 +5,6 @@
     Your contribution goes a long way!
     <img class="float-right" src="{% static "v2/images/diamonds_high_fiving.gif" %}" >
   </p>
-  {% if not show_only_estimate %}
```

Comment:
should we replace this with `clr_active` ? 
https://github.com/gitcoinco/web/blob/382235c40c0874c9b683d2241ec10442ab7153a9/app/grants/views.py#L63